assets/js/extending.js

```
@@ -117,14 +117,13 @@ function showLoginModal(callback) {
 /**
  * Add Authorization header to protected requests
  *
+ * @deprecated This function is no longer used. Authentication is now handled
+ *             via HttpOnly cookies with credentials: 'include' in fetch requests.
  * @param {Headers} headers - Headers object to modify
- * @returns {Headers} Modified headers with Authorization token
+ * @returns {Headers} Unmodified headers (no-op for backwards compatibility)
  */
 function addAuthHeader(headers) {
```

ESLint

[error] 'addAuthHeader' is defined but never used.

(no-unused-vars)